hypertrace-alert-engine/src/main/java/org/hypertrace/alert/engine/RuleEvaluationJob.java

```diff
@@ -20,6 +28,14 @@
 
 public class RuleEvaluationJob implements Job {
   private static final Logger LOGGER = LoggerFactory.getLogger(RuleEvaluationJob.class);
+  private static final ConcurrentMap<String, Counter> alertProcessingErrorCounter =
+      new ConcurrentHashMap<>();
+  private static final String ALERT_PROCESS_ERROR_COUNTER =
```

Comment:
These metrics look fine to me.
- total number of errors during each run
- avg time taken for executing rule (group by tenant)